src/index.ts

```diff
@@ -38,4 +38,8 @@ export {
   resolveApiModel
 } from "./common/parser";
 
+export { Api } from "./generate/api";
```

Comment:
How about having index.ts inside generate directory similar to diff and lint

Reply:
done